test/functions.php

```diff
@@ -30,14 +31,41 @@ public function testextractDataAndDisplayFail()
         $expected_fail = '<section class="container-box"></section>';
         $result_fail = extractDataAndDisplay($input_fail);
         $this->assertEquals($expected_fail, $result_fail);
-
     }
 
     public function testextractDataAndDisplayMalformed()
     {
         $input_malform = "";
         $this->expectException(TypeError::class);
         extractDataAndDisplay($input_malform);
+    }
+
+    public function testisFormCorrectSuccess()
+    {
+        $input1 = 'string';
+        $input2 = '1';
+        $input3 = 'string';
+        $input4 = 'string';
+        $expected_success = '';
+        $result_success = isFormCorrect($input1, $input2, $input3, $input4);
+        $this->assertEquals($expected_success, $result_success);
+    }
 
+    public function testisFormCorrectFail()
```

Comment:
Optional, I will pass this code review without it, but I would usually recommend having a single test for each empty field. i.e. one with `isFormCorrect($good_input, $good_input, $good_input, $empty_input);` another for `isFormCorrect($good_input, $good_input, '0', $good_input);` etc.

Reply:
Thanks you, I'll bear this in mind for the rest !